updater/versions/update-8.py

```diff
@@ -0,0 +1,181 @@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@OCO@@@@@@@@@@@@8CC8@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@[0m
+[36m@@Co@@@@@@@@@@@@@@@@@@@@@@@@@:O@@@@@@O:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@oc@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@O.@@@@@@@@@@@@@@@@@@@@oc@@@@@@@@@@@@@@@@@@@
+@@oc@@@@@@@O:coo:C@@:ccoo:o@@.O@O:coo::@@O.ooo:O@@:::oc:c@o::c8@c:co::O@c::::8@@
+@@oc@@@@@@8.@@@@@co@:C@@@@:C@:OO.@@@@O.@C:@@@@@cC@cc@@@@:c@oc@@::@@@@8:8C:O@@@@@
+@@oc@@@@@@O:888888@@:C@@@@co@:OC:@@@@O.@oc@@@@@co@:c@@@@cc@cc@@::OOOOOO@C:8@@@@@
+@@o:888888@cc@@@Cc@@::O@@C.@@:O@:o@@O::@@:o@@@o:@@c:o@@C:C@o:@@O:o@@8co@C:8@@@@@
+@@@@@@@@@@@@@@88@@@@:C@88@@@@@@@@@88@@@@@@@@8@@@@@cc@OO8@@@@8C8@@@OCO@@@@8@@@@@@
+@@@@@@@@@@@@@@@@@@@@:C@@@@@@@@@@@@@@@@@@@@@@@@@@@@:c@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@[0m
+"""
+
+BOOT_CONFIG = """
+# Minimal (with no vc_vchi_sm_init failure) GPU memory for headless mode
+gpu_mem=32
+# Enable the hardware watchdog
+dtparam=watchdog=on
+# Turn power LED into heartbeat
+dtparam=pwr_led_trigger=heartbeat
+# Enable the hardware random number generator (RNG)
+dtparam=random=on
+"""
+WATCHDOG_CONF_PATH = "/etc/watchdog.conf"
+SUDOERS_CONF_PATH = "/etc/sudoers"
+MOTD_HEAD_PATH = "/etc/motd.head"
+BOOT_CONFIG_PATH = "/boot/config.txt"
+LEPIDOPTER_VERSION_PATH = "/etc/lepidopter_version"
+
+def write_watchdog_conf():
+    with open(WATCHDOG_CONF_PATH, "w") as out_file:
+        out_file.write(WATCHDOG_CONF)
+
+def write_sudoers_conf():
+    with open(SUDOERS_CONF_PATH, "w") as out_file:
+        out_file.write(SUDOERS_CONF)
+
+def write_motd_head():
+    with open(MOTD_HEAD_PATH, "w") as out_file:
+        out_file.write(MOTD_HEAD)
+
+def write_boot_config():
+    with open(BOOT_CONFIG_PATH, "w") as out_file:
+        out_file.write(BOOT_CONFIG)
+
+def _perform_update():
+    check_call(["apt-get", "-q", "update"])
+    # Install OpenVPN package without starting the OpenVPN service
+    check_call(["apt-get", "-y", "install", "openvpn"], env={"RUNLEVEL": "1"})
+    # Install newest e2fsprogs due to incompatibly with ext4 file system checks
+    check_call(["apt-get", "-y", "install", "-t", "jessie-backports", "e2fsprogs"])
+    # Install obfs4proxy (includes a lite version of meek)
+    check_call(["apt-get", "-y", "install", "-t", "stretch", "obfs4proxy"])
+    # Remove old system versions of python-openssl
+    check_call(["apt-get", "-y", "remove", "python-openssl"])
+
+    # Remove bcm2708_rng entry as modules are now handled by /boot/config.txt
+    check_call(["cp", "-b", "/etc/modules", "/etc/modules.bak"])
+    check_call(["sed", "-i",  "'/bcm2708_rng/d'", "/etc/modules"])
+
+    # Disable OpenVPN systemd unit
+    check_call(["systemctl", "disable", "openvpn"])
+
+    # Mount option noatime disables file access writes every time a file is read
+    check_call(["sed", "-i", "'s/\/ ext4/\/ ext4 defaults,noatime/'", "/etc/fstab"])
+
```

Comment:
You should also upgrade `txtorcon` to `0.19.1` so that a) It matches what is in the lepidopter image b) If by chance somebody ran an upgrade of lepidopter in the 6 day window that txtorcon was broken in ooniprobe it will get fixed.